### DISCUSSION TITLE
Slow scroll speed causes hitobjects to "load in" on the playfield

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#16439] [Q&A] [answered]
I've been searching for a long time for a ranked map that displays this behavior, and I've finally found one: https://osu.ppy.sh/b/2862238:

https://user-images.githubusercontent.com/74262407/149232391-75546f0c-ee46-4f65-9463-a404512a297b.mp4

The notes in the kiai section load onto the playfield due to their scroll speed. The lowest seen is 0.14x 110BPM which doesn't even hit the old minimum of 0.1x let alone the new 0.01x minimum.
Additionally, this behavior can be easily observed in some slower scroll speed taiko maps when applying Difficulty Adjust with 0.25 scroll speed (this example uses https://osu.ppy.sh/b/1803863):

https://user-images.githubusercontent.com/74262407/149234860-7e46df80-129b-4f57-a183-946bad18532b.mp4

Version 2022.108.0

**Comment by smoogipoo** [comment, marked as answer]
Weird, it's supposed to account for this... https://github.com/ppy/osu/issues/16440
